source/AI.cpp

```diff
@@ -1095,8 +1123,13 @@ void AI::AskForHelp(Ship &ship, bool &isStranded, const Ship *flagship)
 			if((helper->IsYours() && !ship.IsYours()) || helper.get() == flagship)
 				continue;
 			// Your escorts should not help each other if already under orders.
```

Comment:
I'll take a look and update

Reply:
I went through git blame to try to see why I did this.  Here's what I found.

> If your own escorts become disabled your mining fleet should prioritize repairing escorts instead of mining.